packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -97,11 +100,14 @@ export function CrossmintAuthProvider({ embeddedWallets, children, appearance }:
         return "logged-out";
     };
 
-    const fetchAuthMaterial = async (refreshToken: string): Promise<AuthMaterial> => {
-        const authMaterial = await crossmintAuthService.refreshAuthMaterial(refreshToken);
-        setAuthMaterial(authMaterial);
-        return authMaterial;
-    };
+    const fetchAuthMaterial = useCallback(
+        async (refreshToken: string): Promise<AuthMaterial> => {
+            const authMaterial = await crossmintAuthService.refreshAuthMaterial(refreshToken);
+            setAuthMaterial(authMaterial);
+            return authMaterial;
+        },
+        [crossmintAuthService, setAuthMaterial]
```

Comment:
do we need these as dependencies? May be required by linter, but it's doesn't seem to be the case that if these vars change, we need to re-run the function

Reply:
rather, is there a benefit of using `useCallback` here instead of what we previously had? 
Is it preventing some memory leaks to hold references to the service / `set` method?

Reply:
Yup, better to remove the deps here. Will do